README.md

```diff
@@ -28,8 +28,96 @@ The auth-bundle can be installed using Giant Swarm's web interface or via direct
 
 ## Configuring
 
+The auth-bundle can be configured in 2 different ways: "Default configuration" and "Full configuration". These 2 ways can be combined if needed.
+
+### Default configuration
+
+Default configuration minimizes manual setup and performs the most of the auth-bundle apps' configuration automatically.
+You only need to provide the configuration of Dex connectors you intend to use and basic rbac setup. 
+Everything else is taken care of automatically.
+
+Default configuration is suitable for basic use cases, which do not require any fine-tuning.
+
+#### Config properties
+
+```yaml
+# Optional: Default configuration root object
+defaultConfig:
+
+  # Optional: Determines whether the K8s Authenticator is deployed together with Dex
+  deployDexK8SAuthenticator: false
+
+  # Required: In case the defaultConfig is defined, the oidc property is required
+  oidc:
+
+     # Optional: Defines expiration rules for tokens generated by Dex
+     expiry:
+        signingKeys: 6h
+        idTokens: 30m
+
+     # Required: Specification of Dex connectors
+     customer: 
+        connectors:
+           - id: CONNECTOR_ID # common name is "customer"
+             connectorName: CONNECTOR_NAME
+             connectorType: CONNECTOR_TYPE # e.g. "microsoft", "github", etc.
+             connectorConfig: |- # specific configuration based on connector type, structure comes from Dex.
+                clientID: CLIENT_ID
+                clientSecret: CLIENT_SECRET
+                tenant: TENANT_ID
+                redirectURI: CALLBACK_URL # optional: if omitted, it will be provided automatically
+
+  # Optional but recommended: RBAC setup
+  rbac:
+  - role: ROLE_NAME # required
+    namespaces: # optional: if omitted, cluster-wide bindings are created
+    - NAMESPACE
+    groups: # either groups or users or both are required
+    - CUSTOMER_GROUP_NAME
+    users:
+    - USER_NAME
+```
+
+Ensure that you [enable the required OIDC settings on the cluster resource](https://docs.giantswarm.io/advanced/access-management/configure-dex-in-your-cluster/#configure-the-oidc-values-on-the-cluster-resource).
+
+Most Dex connectors require credentials to be provided in the configuration. 
+Therefore, it is recommended to [include the default configuration in a secret](https://docs.giantswarm.io/vintage/getting-started/app-platform/app-configuration/#example-secret) when installing the auth-bundle.
+
+Connector configuration (`defaultConfig.oidc.customer.connectors[n].connectorConfig`) format depends on the connector type (`defaultConfig.oidc.customer.connectors[n].connectorType`). 
+All supported connector types and configuration formats can be found in [Dex documentation](https://dexidp.io/docs/connectors/).
+
+#### Example configuration
+
+This example shows the minimum required configuration.
```

Comment:
Maybe we can specify that it's the minimum required configuration for azure ad

Reply:
Adjusted the text to say the following (incl. the link to Dex documentation):
This example shows the minimum required configuration for [Microsoft connector in Dex](https://dexidp.io/docs/connectors/microsoft/).